src/Validator/DateRange.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace AppBundle\Validator;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"CLASS", "ANNOTATION"})
+ */
+class DateRange extends Constraint
+{
+    public $startDateField;
+    public $endDateField;
+    public $interval;
+    public $message = 'The end date can not exceed {{ limit }}';
```

Comment:
Can you use translation here ?

Reply:
Yes for the built-in validators, but IMHO it's more consistent and easly to use translations for our validators messages, for example : https://github.com/EnMarche/en-marche.fr/blob/master/src/Validator/GeocodableAddress.php#L17

Reply:
I check them all and it's not all of them which use translation string so I let you deal with @ottaviano :p
But yea you made a point, it would be easier to use with a default message which matches a translation.

Reply:
not big deal, replaced by key 😗 @thomasbisignani